<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-08-14T06:14:54Z" name="Suite" finished-at="2023-08-14T06:14:57Z" duration-ms="3206">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:org.runner.TestRunnerClass@4397ad89]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2023-08-14T06:14:54Z" name="Test" finished-at="2023-08-14T06:14:57Z" duration-ms="3206">
      <class name="org.runner.TestRunnerClass">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:org.runner.TestRunnerClass@4397ad89]" started-at="2023-08-14T06:14:54Z" name="setUpClass" finished-at="2023-08-14T06:14:54Z" duration-ms="130" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@42714a7]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:org.runner.TestRunnerClass@4397ad89]" started-at="2023-08-14T06:14:54Z" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-08-14T06:14:57Z" duration-ms="3054" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Validation of Facebook with invalid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Validation of Facebook application"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at org.steps.StepDefintionClass.user_clicks_on_the_login_button(StepDefintionClass.java:30)
at âœ½.User clicks on the login button(file:///C:/Users/LENOVO/newEclipseWorkspe/CucumberTestng/src/test/resources/Facebook.feature:5)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:org.runner.TestRunnerClass@4397ad89]" started-at="2023-08-14T06:14:57Z" name="tearDownClass" finished-at="2023-08-14T06:14:57Z" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- org.runner.TestRunnerClass -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
